source/elements/oneMKL/source/domains/rng/device_routines/device-engines.rst

```diff
@@ -0,0 +1,42 @@
+.. SPDX-FileCopyrightText: 2023 Intel Corporation
+..
+.. SPDX-License-Identifier: CC-BY-4.0
+
+.. _onemkl_device_rng_engines:
+
+Device Engines (Basic Random Number Generators)
+===============================================
+
+oneMKL RNG provides two device pseudorandom number generators:
```

Comment:
Two? I see four below. It's better to avoid such numbers here to support it easier in future

Reply:
Yep, thanks. I've made task for myself to change it in devref, but smh missed it here